README.md

```diff
@@ -130,8 +133,8 @@ Imports a **Daru::DataFrame** from a **.csv** or **.csv.gz** file.
     require 'daru/io/importers/csv'
 
     #! Usage from Daru::IO
-    df1 = Daru::IO::Importers::CSV.new('path/to/file.csv', skiprows: 10, col_sep: ' ').call
-    df2 = Daru::IO::Importers::CSV.new('path/to/file.csv.gz', skiprows: 10, compression: :gzip).call
+    df1 = Daru::IO::Importers::CSV.new(skiprows: 10, col_sep: ' ').read('path/to/file.csv')
```

Comment:
Funny thing here (which we'll probably postpone, considering how small time we have) is it looks _absolutely, totaly, astonishingly_ unnatural. 

If we'll think about importer as an _object_, usable on itself, in my head the following is logical:
```ruby
importer = Daru::IO::Importers::CSV.read('mydata.csv', col_sep: ' ') # reads once from file, converts data to intermediate representation
# or
importer = Daru::IO::Importers::CSV.parse(some_string_from_other_source, col_sep: ' ')
df1 = importer.call(skiprows: 10, count: 20) # first 10 lines to one DF, quickly
df2 = importer.call(skiprows: 20) # the rest to another one, quickly
```

This way we know why we created the object and what we've passed to constructor. 
But what is the object created with `Daru::IO::Importers::CSV.new(skiprows: 10, col_sep: ' ')`, what's its "physical meaning" or "semantics"?.. ¯\\\_(ツ)_/¯

Does it make sense for you?..

Reply:
The main idea I had behind the `Importers.new(opts).read(path)` was so that it'd facilitate in automating if the Importer is used to collect similar data from LOTS of sources, For example, a folder containing lots of csv files with maybe some daily trends in each file.

That way, the `opts` (options) being initialized can be re-used for multiple sources with passing for each call like `Importers::CSV.read(path).call(opts)`. This was what I had in mind while having a semantic like `Importers::CSV.new(opts).read(path)`.

```ruby
csv_files        = Dir.entries('contains/1000/csvs').keep_if { |file| file.end_with?('.csv')}
csv_importer = Daru::IO::Importers::CSV.new(skiprows: 10, col_sep: ' ')

csv_files.each do |filename|
    csv_importer.read(filename)
end.reduce(:concat)
```